homework.md

```diff
@@ -4,110 +4,109 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Used to make website style.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used for developing dynamic websites.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+When people want to share,manage and version control of their code they can use Git & Github.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You take a copy of that repository to your profile.
 
-## Question 10 
+## Question 10
 
 What happens when you clone a repostory?
```

Comment:
Dear Hatice, in my opinion, you have made the answers to 9 and 10 sound practically the same, when there is indeed a difference.
Also what do you mean by 'your profile'?
So I think the answer to 9 is vague.
I recommend have a look at this link
https://www.toolsqa.com/git/difference-between-git-clone-and-git-fork/ 
and then explain a little more regarding your answers to 9 and 10

Reply:
Thank you for your comments. I meant my Github profile. I am not quite sure if the answers are the same. Thanks Delroy.

Reply:
As you said @Hatice06:

- a 'fork' makes a copy of a repository in your **GitHub profile** (online)
- a 'clone' makes a copy of a repository on your **local machine** (on your laptop)

Reply:
Thanks Ellie.